testsuite/run_sets/secondary_finishing.yml

```diff
@@ -4,13 +4,10 @@
 # feature files in testsuite/docs/Guidelines.md in "Rules for features" chapter,
 # as well as guidelines about idempotency in "Idempotency" chapter.
 
+## Finishing Secondary features  BEGIN ##
 
-## Post run features BEGIN ##
+# this feature is disruptive for other features, so it is better at the end
```

Comment:
Shouldn't our tests be isolated? The outcome of one test case should not impact the execution or results of others no matter what.

Reply:
The first reason of this change is to not  executed test in our teardown but only have the checks. Currently if core failed, secondary will not be executed but finishing yes. And we have two heavy tests in this finishing

Reply:
And putting those tests at the end of secondary is not a good idea and we can't run them in parallel

Reply:
Creating numerous certificates for a specific use case is usually manageable, and potential chaos can be avoided by implementing effective cleaning mechanisms. Although my perspective may be biased as I am going through this codebase for the first time, developing a mocking mechanism for this behavior (which is typical) could address parallelization concerns, especially with a number of cores available. Organizing these prolonged tasks in this manner would be a more efficient approach.

Reply:
Again this PR is not a fix on the potential certificate issue we can have if rename server is failing but it's about not executing those two tests during our teardown. We currently have the essential testing failing in our pipelines. It doesn't make sense to run those tests if core is failing. This PR is limited to just that. I don't touch on which order those tests are executed. I just make sure they are not executed if core fail.

Reply:
I think your proposition needs several servers running for the same pipeline. It would indeed be an awesome improvement but from my understanding we are a little bit limiting on the infra side.

Reply:
In my humble opinion, we better keep the scope of this PR limited to fix our current problem.
We can improve it, following this discussion, in a new card appended in our inbox, to address when we have less pressure.

Reply:
Alright, I've identified areas for future improvements. The current process of manually turning off certain elements when a core test fails needs refinement. Ideally, this should be automated through a (piped) pipeline, something like that:
<img width="339" alt="pipelines_graph_dependency_view_links_v13_12" src="https://github.com/uyuni-project/uyuni/assets/36889107/08c68510-4bf8-4760-bbd6-c783b759af25">

I plan to propose a corresponding PR after reviewing our Jenkins items. That thing exploration may be beneficial further on in terms of containerization.

Closing by now.